src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -197,7 +197,7 @@ bool emberAfGroupsClusterViewGroupCallback(app::CommandHandler * commandObj, con
 struct GroupMembershipResponse
 {
     // A capacity of 0xFF means that it is unknown if any further groups MAY be added.
-    static constexpr uint8_t kCapacityUnknown = 0xff;
+    const chip::app::DataModel::Nullable<uint8_t> kCapacityUnknown = chip::app::DataModel::Nullable<uint8_t>(0xff);
```

Comment:
This is wrong.  Per spec unknown means null, not 0xFF.

In fact, as far as I can tell encoding this (`Nullable<uint8_t>(0xff)`) should be failing.  Are we not actually testing this case?

Reply:
Ah, we are in fact testing it, looks like, and those tests are failing....

Reply:
Unknown used to be 0xff, but it is now NULL. Fixed.